implementation/src/main/java/io/smallrye/openapi/runtime/scanner/OpenApiAnnotationScanner.java

```diff
@@ -350,7 +350,7 @@ private OpenAPIImpl jaxRsApplicationToOpenApi(ClassInfo applicationClass) {
         ////////////////////////////////////////
         AnnotationInstance appPathAnno = JandexUtil.getClassAnnotation(applicationClass,
                 OpenApiConstants.DOTNAME_APPLICATION_PATH);
-        if (appPathAnno == null) {
+        if (appPathAnno == null || config.applicationPathDisable()) {
             appPathAnno = JandexUtil.getClassAnnotation(applicationClass, OpenApiConstants.DOTNAME_PATH);
         }
```

Comment:
Is the intent that `@Path` on an `Application` instance would not be impacted by the new configuration?

Reply:
Yeah I wasn't sure about that.  But that's the decision I ended up going with.  What do you think?

Reply:
I think that's good. It keeps the configuration name in alignment with the annotation.